src/relay/backend/contrib/cmsisnn/relay_to_tir.cc

```diff
@@ -164,21 +156,15 @@ class RelayToTIRVisitor : public MixedModeMutator {
                                         ToArg(dilation_w),   ToArg(dilation_h),    ToArg(clip_min),
                                         ToArg(clip_max)};
 
-    // cmsis_nn_dims *input_dims (NHWC)
+    // layout NHWC
```

Comment:
Sorry Ashutosh, I dont still follow what this comment mean. How is this related to the following line ?

Reply:
This was to help understand that the following shape is in NHWC layout which will be passed on to CMSIS-NN API eventually.

Reply:
I think might worth saying that explicitly :)

Reply:
Ack